.github/workflows/scripts/common.sh

```diff
@@ -49,7 +49,8 @@ gh_summary() {
 }
 export -f gh_summary
 
-if [ "$#" -gt 0 ]; then
+# bash trick to check if the script is sourced.
+if ! (return 0 2>/dev/null); then # called
```

Comment:
When using dark magic, maybe we could wrap it in a function to make it easier to understand?


```bash
function is_sourced() {
    (return 0 2>/dev/null)
}
```